news/2024/2024-08-12-the-followpoint-motion-the-ultimate-catcher.md

```diff
@@ -0,0 +1,294 @@
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/news/2024-08-12-the-followpoint-motion-the-ultimate-catcher/avatar-MegaMix.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>
+
+Now that you've reached #1 rank in osu!catch, do you have any other goals in mind? Is there anything else you'd like to achieve now, be that for example an SS on a hard map, or maybe even 2000pp score?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/3885626" style="background-image: url('/wiki/shared/news/2024-08-12-the-followpoint-motion-the-ultimate-catcher/avatar-Motion.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/3885626">Motion</a></p>
+
+Unfortunately, any specific goals don't come to my mind right now. but I will keep pushing myself to set any crazy scores!
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/news/2024-08-12-the-followpoint-motion-the-ultimate-catcher/avatar-MegaMix.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>
+
+And before we finish, let me ask you some bonus questions, starting with the classic one: What plays, be they yours or someone else’s, you find the most impressive? If we pick one of yours, which one are you the most proud of?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/3885626" style="background-image: url('/wiki/shared/news/2024-08-12-the-followpoint-motion-the-ultimate-catcher/avatar-Motion.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/3885626">Motion</a></p>
+
+If I were to pick the most impressive score, it would be probably ["Oracion(TV-Size) [Crystal's Rain]" +HDDTHR](https://osu.ppy.sh/scores/fruits/173627724) played by [ExGon](https://osu.ppy.sh/users/214187), because high approach rate (mostly AR10.5~11) in osu!catch is really tricky and difficult skillset to master.
+
+Also there are a few candidates that are coming to my mind, but if I have to pick one of them, it will be ["Sendan Life (katagiri Bootleg) [World End.]"](https://osu.ppy.sh/scores/fruits/214502143) without a doubt, because this map literally requires most of skillsets which are "consistency", "speed", "stamina" and also to quote ExGon, [“probably the best nomod score for ctb maps, no one will be able to get close for a while”](https://twitter.com/ExG0n/status/1713871203314868578). So I do believe this is enough to be picked.
```

Comment:
```suggestion
Also there are a few candidates that are coming to my mind, but if I have to pick one of them, it will be ["Sendan Life (katagiri Bootleg) [World End.]"](https://osu.ppy.sh/scores/fruits/214502143) without a doubt, because this map literally requires most of skillsets which are "consistency", "speed", "stamina" and also to quote ExGon, [“probably the best nomod score for osu!catch maps, no one will be able to get close for a while”](https://twitter.com/ExG0n/status/1713871203314868578). So I do believe this is enough to be picked.
```